src/utils/NotificationUtils.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+import * as admin from 'firebase-admin';
+// eslint-disable-next-line import/no-unresolved
+import { getMessaging } from 'firebase-admin/messaging';
+
+const schedule = require('node-schedule');
+
+function sendNotification(deviceToken: string, notifData) {
+  const message = {
+    data: notifData,
+    token: deviceToken,
+  };
+  if (deviceToken !== '') {
+    getMessaging().send(message)
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
```

Comment:
Maybe raise an exception here or log if there is no deviceToken provided, will help others debug their code if it gets there